package.json

```diff
@@ -9,20 +9,27 @@
         "url": "git+https://github.com/workleap/wl-web-configs.git"
     },
     "scripts": {
-        "build": "pnpm -r build",
+        "postinstall": "pnpm build",
+
+        "build": "pnpm build:eslint && pnpm build:tsup && pnpm --filter ./packages/* -r build",
         "build:eslint": "pnpm --filter=@workleap/eslint-plugin build",
+        "build:tsup": "pnpm --filter=@workleap/tsup-configs build",
 
         "test": "jest",
 
         "lint": "pnpm run /^lint:.*/",
-        "lint:eslint": "pnpm build:eslint && eslint . --max-warnings=1 --cache --cache-location node_modules/.cache/eslint",
```

Comment:
Removing the build section causes ossues in dev. On a clean install, if you lint before building, it will throw errors

Reply:
I moved it to the "postinstall" script